build-scripts/ios/gen_build_config.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const config = require('../../config.json');
```

Comment:
If we get this from the config file, we move the problem to not to commit the secrets to the config file, should it be some kind of extra file that people may or not in their system, or environment variable as we will need to do it for travis?